tests/rule-engine/fc-config.scm

```diff
@@ -1,10 +1,21 @@
 ;
 ; Configuration file for the example crisp rule base system (used by
 ; fc.scm)
-;
-; Before running any inference you must load that file
 
-; Load the rules
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Load required modules and utils ;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(use-modules (opencog))
+(use-modules (opencog rule-engine))
+
+(load-from-path "utilities.scm")
+(load-from-path "av-tv.scm")
```

Comment:
After saying `(use-modules (opencog))`, one should not need to also load "utilities.scm" and "av-tv.scm" since everything in those files should already be publicly exported by the main opencog module.